### ISSUE TITLE
Quick toggle

### ISSUE THREAD

**Issue by po5** [Andrews54757/Anti-Anti-Debug#1]
I'd like a way to disable it until it's actually needed, as this messes with debugging and other functions the user needs to work reliably in most cases.  
Having it off by default (or just remembering the off state across sessions), and being able to toggle it by just clicking on the pinned extension icon would be a lot nicer than going to `chrome://extensions` and toggling it there.

**Comment by Andrews54757** [comment]
Yeah thats a good idea, early on I just assumed that the global extension toggle would be good enough, but I can see how its slightly more annoying than a icon click. Looking at the [documentation for dynamic content scripts](https://developer.chrome.com/docs/extensions/mv3/content_scripts/#dynamic-declarative), it looks like I need to have a background service worker. Firefox [is weird](https://stackoverflow.com/questions/75043889/manifest-v3-background-scripts-service-worker-on-firefox) when it comes to those and Manifest V3. Gotta setup a build system.

**Comment by Estecka** [comment]
On firefox, there's already something akin to a toogle. Was this your doing?

The extension doesn't work right out of the box when installed, it needs to be explicitely allowed to run specific domains. The permissions can be granted and revoked easily from the extension menu.
![Screenshot_4](https://github.com/Andrews54757/Anti-Anti-Debug/assets/39855800/186248c8-dd55-4827-b9d5-9c2fefea00fd)
![Screenshot_5](https://github.com/Andrews54757/Anti-Anti-Debug/assets/39855800/466e237c-faf4-4cab-884f-ff5087343bbf)
(By the way, this might be worth mentionning in the troubleshooting tips. Made me rage a bit until I figured this was a thing.)
